python/sglang/srt/layers/rotary_embedding.py

```diff
@@ -719,12 +722,6 @@ def forward(
         offsets: Optional[torch.Tensor] = None,
     ) -> Tuple[torch.Tensor, torch.Tensor]:
         """PyTorch-native implementation equivalent to forward()."""
-        query_rot = query[..., : self.rotary_dim]
```

Comment:
query equals to query_rot in deepseek v2, I keep this code here for a reference.